docs/getting-started/installation.md

```diff
@@ -39,12 +39,13 @@ We suggest the following two ways to install Go. Check out the [official docs](h
 
 **Ubuntu:**
 
-At the time of this writing, the latest release is `1.18.10`. We're going to download the tarball, extract it to `/usr/local`, and export `GOROOT` to our `$PATH`
+At the time of this writing, the latest release is `1.2.10`. We're going to download the tarball, extract it to `/usr/local`, and export `GOROOT` to our `$PATH`
```

Comment:
version downloaded below is `1.20.3`, you mention `1.2.10` in the comment above, which is incorrect.